packages/validators/src/db/index.ts

```diff
@@ -4,7 +4,8 @@ import { username } from "../common";
 
 export const ReviewSchema = z.object({
   userId: z.string().min(0).max(255),
-  restaurantId: z.number().int(),
+  restaurantId: z.string(),
+  name: z.string(),
```

Comment:
```suggestion
  restaurantName: z.string(),
```
Try and restrict the value here as well (min and max lengths)

Reply:
done